File: exercice.py
for i in range(1, 101):
    if i % 3 == 0 and i % 5 == 0:
        print("ChausettesSales")
    elif i % 3 == 0:
        print("Chausettes")
    elif i % 5 == 0:
        print("Sales")
    else:
        print(i)

File: exercice.js
for (let i = 1; i <= 100; i++) {
    if (i % 3 === 0 && i % 5 === 0) {
        console.log("ChausettesSales");
    } else if (i % 3 === 0) {
        console.log("Chausettes");
    } else if (i % 5 === 0) {
        console.log("Sales");
    } else {
        console.log(i);
    }
}